libsolidity/ast/Types.cpp

```diff
@@ -2308,9 +2310,10 @@ MemberList::MemberMap FunctionType::nativeMembers(ContractDefinition const*) con
 	case Location::Bare:
 	case Location::BareCallCode:
 	case Location::BareDelegateCall:
+	case Location::Transfer:
 	{
 		MemberList::MemberMap members;
-		if (m_location != Location::BareDelegateCall && m_location != Location::DelegateCall)
+		if (m_location != Location::BareDelegateCall && m_location != Location::DelegateCall && m_location != Location::Transfer)
```

Comment:
It's still different. Humans can think about the gas and take external information into account, whereas the gas can be hard-coded here. Are there actual use cases of supplying max gas (nothing apart from 0 and max makes sense, I guess)?